### PR TITLE
Created the UI to get the list of members in a channel.

### PR DESCRIPTION
**Pull request by Xclusivecyborg**
I created the UI,  I also refactored the shared widgets to make the default search bar look exactly like the one in the design then I added a gesture detector in the third section file to navigate to the page i created.

### CODE REVIEW

File: lib/ui/view/channel/channel_info/widgets/third_section.dart [hunk 1]
```diff
@@ -36,9 +37,17 @@ class ThirdSection extends StatelessWidget {
                   left: 11,
                   top: 19,
                 ),
-                child: Text(
-                  'Members (5)',
-                  style: descriptionStyle(),
+                child: GestureDetector(
+                  onTap: () {
+                    Navigator.push(
```
Comment (isaacadariku) [inline lib/ui/view/channel/channel_info/widgets/third_section.dart:42]: There should be no navigation or logic in your UI code, make this in the view model, use the navigation services provided,

File: lib/ui/view/channel/channel_members/channel_members_list.dart [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                  color: Color(0xFF333333),
+                  // size: 24,
+                ),
+                padding: EdgeInsets.only(left: 18),
+                onPressed: () {},
+                iconSize: 25.0,
+              ),
+              actions: [
+                InkWell(
+                  child: 'Edit'
```
Comment (isaacadariku) [inline lib/ui/view/channel/channel_members/channel_members_list.dart:41]: I thought there is a general style to be use, this won't scale when the time comes. Please use the style guide and remove any dependency on velocityX.

**Review by isaacadariku — CHANGES_REQUESTED** [review]
Nice, make the changes and push the commits. Thank you

File: lib/ui/view/channel/channel_info/widgets/third_section.dart (hunk 1 above)
Comment (Xclusivecyborg) [reply lib/ui/view/channel/channel_info/widgets/third_section.dart]: Changes have been made and commits have been pushed. Kindly take a look

**Review by mayor04 — APPROVED** [review]

**Review by Xclusivecyborg — COMMENTED** [review]
Changes have been made

**Review by isaacadariku — APPROVED** [review]
LGTM

**Review by Quwaysim — APPROVED** [review]

**Pull request merged**